Pcc15

clamytoe opened the pull request:
Umm, hope I did this right...

pybites commented:
cool this seems ok, only moving requirements.txt into your dir

pybites commented:
Ah I basically undid changes to requirements.txt in 15 dir, and I see now that you had your updated requirements.txt already in your user dir, good.

bbelderbos commented:
Tried it locally, this looks awesome, wow. Great job

pybites commented:
Permission to use it for our blog planning? ;)

clamytoe commented:
Thanks! Sure no problem, use it however you like. Should I disable debug mode? Forgot to remove that portion. I should probably add a Readme as well..

pybites commented:
Yep mail them and I will add them to the community branch. Will email you

clamytoe commented:
Sorry,

Ended up just creating a new PR because I added images and modified the app
to add a new feature.

On Fri, Apr 21, 2017 at 4:01 AM PyBites <notifications@github.com> wrote:

> Yep mail them and I will add them to the community branch. Will email you
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pybites/challenges/pull/13#issuecomment-296134675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGztDwl-2IhXoLpjAbQsqxhZp2J46x0Nks5ryHCGgaJpZM4NDydx>
> .
>
-- 
"I have always wanted to be somebody, but I see now I should have been more
specific."
